provisions/hosts.sample

```diff
@@ -11,10 +11,10 @@ jenkins-master
 jenkins-slave
 
 [openshift]
-openshift
+open-shift
```

Comment:
why is it like `open-shift`

Reply:
This is to match the regex and avoid using the name in square brackets `[openshift]`. This will help in vim/editors to replace all the occurrences of `open-shift` with its IP/FQDN, like
```
:%s/open-shift/192.168.1.3/g
```
if I keep it `openshift` and run the regex, the undesired will be replaced too, for eg
```
[192.168.1.3]
192.168.1.3
```
Make sense ?